server/src/test/java/io/whitefox/api/deltasharing/model/FileObjectFileWithoutPresignedUrl.java

```diff
@@ -0,0 +1,129 @@
+package io.whitefox.api.deltasharing.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class FileObjectFileWithoutPresignedUrl {
```

Comment:
Can we make openapi-generator generate this and `server/src/test/java/io/whitefox/api/deltasharing/model/FileObjectWithoutPresignedUrl.java`?
actually they should already be there

Reply:
I don't see this class in the protocol. I can see only the `FileObject` object (which includes the presigned url and the hashed id). The `FileObjectWithoutPresignedUrl` object has been defined to be used only in the testing phase, as it is not possible to control the presigned url and the hashed id. I could also decide to use the real `FileObject` object and ignore the presigned url and id (hashed). However, it seemed cleaner to me to define a class without those two fields, and make the assertion on the whole object.

Reply:
If it's needed only in tests, it's better like this ;) sorry I did not get it earlier